src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig

```diff
@@ -191,7 +191,7 @@
   {{- parent() -}}
 {%- endblock %}
 
-{#{% block checkbox_widget -%}
+{# {% block checkbox_widget -%}
```

Comment:
Maybe you can delete that commented code?

Reply:
Let's think about this in another PR (as it's not related) just in order to event it being blocked